content/advanced/330_servicemesh_using_appmesh/cleanup/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Cleanup"
+date: 2020-01-27T08:30:11-07:00
+weight: 100
+draft: false
+---
+
+#### Delete all the resources in the namespace
+
+```bash
+kubectl delete all --all -n prodcatalog-ns
+```
+
+#### Delete Product Catalog apps
+
+```bash
+kubectl delete namespace prodcatalog-ns
+```
+
+#### Delete the Product Catalog mesh
+
+```bash
+kubectl delete meshes prodcatalog-mesh
+```
+
+#### Uninstall the Helm Charts
+
+```bash
+helm -n appmesh-system delete appmesh-controller
+```
+
+#### Delete AWS App Mesh CRDs
+
+```bash
+for i in $(kubectl get crd | grep appmesh | cut -d" " -f1) ; do
+kubectl delete crd $i
+done
+```
+
+#### Delete the AWS App Mesh namespace
+
+```bash
+kubectl delete namespace appmesh-system
+```
+
+#### Delete the appmesh-controller service account
+```bash
+eksctl delete iamserviceaccount --cluster eksworkshop-eksctl   --namespace appmesh-system --name appmesh-controller
+```
+
+#### Delete the Fargate service account
+```bash
+eksctl delete iamserviceaccount --cluster eksworkshop-eksctl   --namespace prodcatalog-ns --name prodcatalog-sa
+```
+
+#### Delete Fargate profile
+```bash
+eksctl delete fargateprofile \
+  --name fargate-productcatalog \
+  --cluster eksworkshop-eksctl
+```
+
+#### Delete Nodegroup
+```bash
+envsubst < ./deployment/clusterconfig.yaml | eksctl delete nodegroup -f -  --approve
+```
```

Comment:
We also have:

3 ECR repos with images to clean up
We have amazon-cloudwatch and aws-observability namespaces to clean up

Reply:
updated